test/test_scanner.py

```diff
@@ -671,6 +671,22 @@ def _file_test(self, url, filename, package, version):
                     "CVE-2002-0760"
                 ],
             ),
+            (
+                "test-BZIP2-1.0.2.out",
```

Comment:
Please rename this file and change this line to be lowercase, so that it matches the other tests.